### PR TITLE
[15] Research Programming Languages

### PR DESCRIPTION
**Pull request by kamena1994**
## Task: `Research programming languages`
- `added programming language folder and 
text file`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, remove the space in programming language.txt

File: submissions/benjaminKamena/aboutMe/aboutMe.txt.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Am benjamin kamena,
```
Comment (ToluAkin) [inline submissions/benjaminKamena/aboutMe/aboutMe.txt.txt:1]: @kamena1994 This file exist already. Remove it from your pull request. To do that, make sure you are on this branch, delete this file, do `git add .`, commit the changes, then push.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
Why are you deleting all your files

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove the spaces in `Programing Languages 1`. you can replace it with `_` or `-`

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**